pkg/api/v1/helper/helpers_test.go

```diff
@@ -499,3 +500,155 @@ func TestGetAffinityFromPodAnnotations(t *testing.T) {
 		}
 	}
 }
+
+func TestGetPodQOS(t *testing.T) {
```

Comment:
> sure; I'll add a test for internal one.

The key piece bit is that its the same test, not a copy/paste.  That way a change in one will cause the test fail.  A separate test won't achieve the same goal.